src/WebJobs.Extensions.DurableTask/ContextInterfaces/IDurableOrchestrationClient.cs

```diff
@@ -226,6 +226,18 @@ public interface IDurableOrchestrationClient
         /// <returns>A task that completes when the terminate message is enqueued if necessary.</returns>
         Task TerminateAsync(string instanceId, string reason);
 
+        /// <summary>
+        /// Suspends a running orchestration instance.
+        /// </summary>
+        /// <returns>A task that completes when the suspend message is enqueued if necessary.</returns>
+        Task SuspendAsync(string instanceId, string reason);
+
+        /// <summary>
+        /// Resumes a suspended orchestration instance.
+        /// </summary>
+        /// <returns>A task that completes when the resume message is enqueued if necessary.</returns>
+        Task ResumeAsync(string instanceId, string reason);
```

Comment:
Just noting that while I said in the durabletask PR that we shouldn't add methods to interfaces because it's a breaking change, we make an exception to this rule here because 1) we don't support implementations of this interface that we don't own and 2) it would create a really ugly user experience if customers had to cast the interface from one type to another just to do suspend/resume.

Just mentioning this in case you were wondering. No changes required here.